src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"Pink", "Green", "Sky Blue",
+            "Purple", "Yellow", "Mint"};
+
+    public static String getRandomColor() {

```

Comment:
using static on methods is a bad practice, and its not OOP style
```suggestion
    public String getRandomColor() {
```

Reply:
not fixed